### DISCUSSION TITLE
Support For Capacitor

### DISCUSSION THREAD

**Discussion by DevamshManoj04** [talsec/Free-RASP-Community#20] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
Our platform depends on your plugin for security, Since we are migrating to capacitor no alternatives are available.

**Describe the solution you'd like**
Added support for capacitor

**Comment by syakymchuk** [comment]
Hello @DevamshManoj04 ,

There is a short-term plan for Capacitor support in the premium plan RASP+ and AppiCrypt. Currently, we can't provide any committed date when the Capacitor support will be available for the community version. 

Should you have any question regarding RASP+ please don't hesitate to [contact](https://www.talsec.app/contact)

**Reply by syakymchuk** [reply, marked as answer]
@DevamshManoj04, the freeRASP for Capacitor was released. Just, in case you missed this information.
